### PR TITLE
Autopilot update + formatting fixes

### PR DESCRIPTION
**Pull request by FSXAC**

### CODE REVIEW

File: docs/deliverables/deliverables.tex [hunk 1]
```diff
@@ -77,6 +75,7 @@ \subsection{Other Artifacts}
 \begin{itemize}
     \item Documents and Meeting Notes Repository ({\url{https://github.com/Capstone-Skynet/Capstone-Skynet.github.io}})
     \item Source Code Repository ({\url{https://github.com/Capstone-Skynet/Integration}})
+    \item CAD models of hardware and mechanical components
```
Comment (peterdeutsch) [inline docs/deliverables/deliverables.tex:78]: This is already included above

File: docs/design/3_2_multirotor_v2.tex [hunk 2]
```diff
@@ -228,7 +228,7 @@ \subsubsection{Battery and Electrical Systems}
 \end{figure}
 
 \subsubsection{Flight Computer and Sensors}
-To allow for a user with minimal flight training to use the multicopter successfully, a standalone flight controller is integrated into the multicopter system to allow for the automation of a majority of flight operations. 
+To allow for a user with minimal flight training to use the multicopter successfully, a standalone flight controller is integrated into the multicopter system to allow for the automation of a majority of flight operations. The flight controller unit (FCU) are also refered to as ``autopilot'' for their autonomous abilities.
```
Comment (peterdeutsch) [inline docs/design/3_2_multirotor_v2.tex:231]:
```suggestion
To allow for a user with minimal flight training to use the multicopter successfully, a standalone flight controller unit is integrated into the multicopter system to allow for the automation of a majority of flight operations. The flight controller unit (FCU) is also often referred to as ``autopilot'' for its autonomous abilities. 
```

File: docs/design/3_2_multirotor_v2.tex [hunk 3]
```diff
@@ -237,9 +237,8 @@ \subsubsection{Flight Computer and Sensors}
...
+The flight controller included on the multirotor is called ArduPilotMega (APM) 2.8 using firmware of the same name. APM is a ubiquitousopen-source hardware and firmware stack. APM Planner 2.0 is a ``ground station'' software package used to configure the autopilot before flight operations such as installing firmware, or during flight operations to provide real time telemetry \footnote{Telemetry requires additional hardware that is currently not supported on the multirotor}. As seen in Figure \ref{flightdisplay}, APM Planner 2.0 provides users with easy-to-use GUI for tuning multirotor settings and paramters. The choice of flight controller and its dedicated software was necessitated by cost-limitations at purchasing time, however the differences between budget flight controllers is ultimately marginal.
```
Comment (peterdeutsch) [inline docs/design/3_2_multirotor_v2.tex:241]:
```suggestion
The flight controller included on the multirotor is called ArduPilotMega (APM) 2.8 using firmware of the same name. APM is a ubiquitous open-source hardware and firmware stack. APM Planner 2.0 is a ``ground station'' software package used to configure the autopilot before flight operations such as installing firmware, or during flight operations to provide real time telemetry \footnote{Telemetry requires additional hardware that is currently not supported on the multirotor}. As seen in Figure \ref{flightdisplay}, APM Planner 2.0 provides users with easy-to-use GUI for tuning multirotor settings and parameters. The choice of flight controller and its dedicated software was necessitated by cost-limitations at purchasing time, however the differences between budget flight controllers is ultimately marginal.
```

File: docs/deliverables/rev_history.tex [hunk 4]
```diff
@@ -12,7 +12,7 @@ \section*{Revision History}
 1.0 & PD & 2019-11-23 & \texttt{} & Updated header to synchronize styles for Milestone II.\\
 1.1 & PD & 2020-04-06 & \texttt{} & Updated physical computation platform deliverables.\\
 1.2 & PD & 2020-04-08 & \texttt{} & Updated delivery instructions.\\
-
+1.3 & MH & 2020-04-08 & \texttt{} & Final revision for M4.\\
```
Comment (FSXAC) [inline docs/deliverables/rev_history.tex:15]:
```suggestion
1.3 & MH & 2020-04-08 & \texttt{b1680955} & Final revision for M4.\\
```

File: docs/design/rev_history.tex [hunk 5]
```diff
@@ -15,7 +15,7 @@ \section*{Revision History}
 1.2 & MH & 2020-02-09 & \texttt{--} & Updated title page; updated multirotor air frame, propulsion system, flight controller unit, and battery section.\\ 
 1.3 & PD & 2020-04-05 & \texttt{--} & Updated sections on ML data flows, image acquisition/formatting, ML result delay handling, and PMB-Base Station communications.\\ 
 1.4 & MH & 2020-04-08 & \texttt{--} & Updated Multirotor section.\\ 
-
+1.5 & MH/PD & 2020-04-08 & \texttt{} & Final revision for M4.\\
```
Comment (FSXAC) [inline docs/design/rev_history.tex:18]:
```suggestion
1.5 & MH/PD & 2020-04-08 & \texttt{b1680955} & Final revision for M4.\\
```

File: docs/ops_spec/rev_history.tex [hunk 6]
```diff
@@ -11,7 +11,7 @@ \section*{Revision History}
 1.0 & PD & 2019-11-25 &  & Minor update for Milestone II.\\
 1.1 & PD & 2020-02-09 &  & Added preambulatory content and PetaLinux build instructions \\
 1.2 & AW/PD & 2020-04-08 &  & Added FAQs, Troubleshooting, Flight Checklists, and Suggested Upgrades \\
-
+1.3 & MH/PD & 2020-04-08 & \texttt{} & Final revision for M4.\\
```
Comment (FSXAC) [inline docs/ops_spec/rev_history.tex:14]:
```suggestion
1.3 & MH/PD & 2020-04-08 & \texttt{b1680955} & Final revision for M4.\\
```

File: docs/reqs/rev_history.tex [hunk 7]
```diff
@@ -15,7 +15,7 @@ \section*{Revision History}
 1.1 & MH & 2020-02-07 & \texttt{--} & Updated RPAS requirements to reflect constraints.\\
 1.2 & PD & 2020-02-08 & \texttt{--} & Loosened ML latency/throughput requirements with Client approval.\\
 1.3 & PD & 2020-04-08 & \texttt{4c20d244} & Added clarification as to which requirements were removed from the project in light of the COVID-19 pandemic.\\
-
+1.4 & MH/PD & 2020-04-08 & \texttt{} & Final revision for M4.\\
```
Comment (FSXAC) [inline docs/reqs/rev_history.tex:18]:
```suggestion
1.4 & MH/PD & 2020-04-08 & \texttt{b1680955} & Final revision for M4.\\
```

File: docs/validation/rev_history.tex [hunk 8]
```diff
@@ -15,7 +15,7 @@ \section*{Revision History}
 1.1 & AH & 2020-02-09 & & Added Full System Tests \\
 1.2 & MH & 2020-02-09 & & Added PID Tuning and Testing for Multirotor RPAS. \\
 1.3 & MH/PD & 2020-04-08 & & Added Multirotor Validation section. \\
-
+1.4& MH/PD & 2020-04-08 & \texttt{} & Final revision for M4.\\
```
Comment (FSXAC) [inline docs/validation/rev_history.tex:18]:
```suggestion
1.4 & MH/PD & 2020-04-08 & \texttt{b1680955} & Final revision for M4.\\
```

File: docs/deliverables/deliverables.tex (hunk 1 above)
Comment (FSXAC) [reply docs/deliverables/deliverables.tex:78]: Here i meant the actual CAD files

**Pull request merged**